frontend/src/views/c-authorship.vue

```diff
@@ -103,24 +103,20 @@
       .file(v-bind:ref="file.path")
         .title
           span.caret(v-on:click="toggleFileActiveProperty(file)")
-            .tooltip(v-show="file.active")
+            .tooltip(
+              v-show="file.active"
+              v-on:mouseover="onTooltipHover(file.path + 'hide-file-tooltip')",
```

Comment:
Just a minor suggestion. This needs to adopt ES6+ way of string interpolation as is shown [here](https://github.com/reposense/RepoSense/pull/1845/files)